roles/openshift_metrics/templates/hawkular_metrics_rc.j2

```diff
@@ -12,48 +12,64 @@ spec:
   template:
     version: v1
     metadata:
-      name: hawkular-metrics
       labels:
         metrics-infra: hawkular-metrics
         name: hawkular-metrics
     spec:
-      serviceAccountName: hawkular-metrics
+      serviceAccount: hawkular
       containers:
-      - name: hawkular-metrics
-        image: {{image_prefix}}metrics-hawkular-metrics:{{image_version}}
+      - image: {{image_prefix}}metrics-hawkular-metrics:{{image_version}}
+        name: hawkular-metrics
         ports:
-        - containerPort: 8080
-          name: "http-endpoint"
-        - containerPort: 8443
-          name: "https-endpoint"
+        - name: http-endpoint
+          containerPort: 8080
+        - name: https-endpoint
+          containerPort: 8443
+        - name: ping
+          containerPort: 8888
         command:
         - "/opt/hawkular/scripts/hawkular-metrics-wrapper.sh"
         - "-b"
         - 0.0.0.0
-        - "-Dhawkular-metrics.cassandra-nodes=hawkular-cassandra"
-        - "-Dhawkular-metrics.cassandra-use-ssl"
-        - "-Dorg.apache.tomcat.util.buf.UDecoder.ALLOW_ENCODED_SLASH=true"
-        - "-Dorg.apache.catalina.connector.CoyoteAdapter.ALLOW_BACKSLASH=true"
-        - "-Dhawkular-metrics.openshift.auth-methods=openshift-oauth,htpasswd"
-        - "-Dhawkular-metrics.openshift.htpasswd-file=/secrets/hawkular-metrics.htpasswd.file"
+        - "-Dhawkular.metrics.cassandra.nodes=hawkular-cassandra"
```

Comment:
I don't think necessarily unless there is something variable about them.  Previously they were hardcoded into an openshift template so I think its ok to do the same here.